ui/src/app/views/homepage/component.tsx

```diff
@@ -0,0 +1,394 @@
+import {BugReportComponent} from 'app/views/bug-report/component';
+import {environment} from 'environments/environment';
+
+import * as React from 'react';
+
+import {
+  Clickable,
+  styles as buttonStyles
+} from 'app/components/buttons';
+import {ClrIcon} from 'app/components/icons';
+import {profileApi} from 'app/services/swagger-fetch-clients';
+import {reactStyles, ReactWrapperBase, withStyle} from 'app/utils';
+import {navigateByUrl} from 'app/utils/navigation';
+import {
+  BillingProjectStatus,
+  IdVerificationStatus,
+  PageVisit,
+  Profile,
+  ProfileService
+} from 'generated';
+
+
+const styles = reactStyles({
+  mainHeader: {
+    color: '#FFFFFF',
+    fontSize: 28,
+    fontWeight: 400,
+    width: '25.86%',
+    display: 'flex',
+    minWidth: '18.2rem',
+    marginLeft: '4%',
+    marginTop: '4%',
+    letterSpacing: 'normal'
+  },
+  minorHeader: {
+    color: '#FFFFFF',
+    fontSize: 18,
+    fontWeight: 600,
+    display: 'flex',
+    marginTop: '1rem',
+    lineHeight: '24px'
+  },
+  text: {
+    color: '#FFFFFF',
+    fontSize: 16,
+    lineHeight: '22px',
+    fontWeight: 150,
+    marginTop: '3%'
+  },
+  infoBox: {
+    padding: '1rem',
+    backgroundColor: '#FFFFFF',
+    borderRadius: '5px',
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between'
+  },
+  infoBoxHeader: {
+    fontSize: 16,
+    color: '#262262',
+    fontWeight: 600
+  },
+  infoBoxBody: {
+    color: '#000',
+    lineHeight: '18px'
+  },
+  infoBoxButton: {
+    color: '#FFFFFF',
+    height: '49px',
+    borderRadius: '5px',
+    marginLeft: '1rem',
+    maxWidth: '20rem'
+  },
+  error: {
+    fontWeight: 300,
+    color: '#DC5030',
+    fontSize: '14px',
+    marginTop: '.3rem',
+    backgroundColor: '#FFF1F0',
+    borderRadius: '5px',
+    padding: '.3rem',
+  }
+});
+
+export const Error = withStyle(styles.error)('div');
+
+const AccountLinkingButton: React.FunctionComponent<{
+  failed: boolean, completed: boolean, failedText: string,
+  completedText: string, defaultText: string, onClick: Function
+}> = ({failed, completed, defaultText, completedText, failedText, onClick}) => {
+  if (failed) {
+    return <Clickable style={{...buttonStyles.base,
+      ...styles.infoBoxButton, backgroundColor: '#f27376'}} disabled={true}>
+      <ClrIcon shape='exclamation-triangle'/>{failedText}
+    </Clickable>;
+  } else if (completed) {
+    return <Clickable style={{...buttonStyles.base,
+      ...styles.infoBoxButton, backgroundColor: '#8BC990'}} disabled={true}>
+      <ClrIcon shape='check'/>{completedText}
+    </Clickable>;
+  } else {
+    return <Clickable style={{...buttonStyles.base,
+      ...styles.infoBoxButton, backgroundColor: '#2691D0'}}
+                      onClick={onClick}>
+      {defaultText}
+    </Clickable>;
+  }
+};
+
+export interface AccountLinkingProps {
+  eraCommonsLinked: boolean;
+  trainingCompleted: boolean;
+}
+
+export class AccountLinking extends
```

Comment:
Naming nitpick: "Account Linking" doesn't quite capture that this component will also show compliance training completion steps required for access to the workbench.

And will we also use this component to show users some information when their account has been disabled – or do they just see a straight 403? (Same goes w/ ID Verification, which will probably become a blocking attribute that replaces ERA commons & will need to be included in this screen.)

Possible alternatives: "UserLockout", "WorkbenchAccessDenied", "AccessDeniedView", "StepsRequiredForAccess"?

@calbach for thoughts.

Reply:
Agreed. Naming is hard, I don't have a great recommendation here, another option: UnregisteredSection